home.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="home page">
+    <link rel="stylesheet" href="dist/css/style.css">
+    <script src="https://kit.fontawesome.com/059c84c64c.js" crossorigin="anonymous"></script>
+    <title>Home</title>
+</head>
+<body class="bg-light">
+    <nav class="navbar bg-info">
+        <div class="container-fluid">
+            <a href="home.html" class="navbar-brand fs-3">Social network</a>
+            <div id="navbarNavAltMarkup">
+                <div class="navbar-nav d-flex flex-row">
+                    <a class="nav-link mx-3 fs-5 active" aria-current="page" href="home.html">Home</a>
+                    <a class="nav-link mx-3 fs-5" href="profile.html">Profile</a>
+                </div>
+            </div>
+            <form class="d-flex">
+                <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+                <button class="btn btn-success" type="submit">Search</button>
+            </form>
+        </div>
+    </nav>
+    <div class="container">
+        <div class="row">
+            <h1 class="d-flex justify-content-center my-3">Share your thoughts</h1>
+            <div class="col-xs-12 col-md-4 col-lg-4 flex-shrink">
```

Comment:
```suggestion
            <div class="col-xs-12 col-md-4 col-lg-4">
```

flex-shrink looks out of place to me